lively.morphic/text/label.js

```diff
@@ -214,6 +214,7 @@ export class Label extends Morph {
     super(obj.dissoc(props, ['fontMetric']));
     if (fontMetric) { this._fontMetric = fontMetric; }
     this._cachedTextBounds = null;
+    this.acceptsDrops = false;
```

Comment:
Actually the canonical way to achieve this is to set the defaultValue to `false` in the property definition of acceptsDrops.